l10n_ch_pain_base/models/account_move_line.py

```diff
@@ -1,21 +1,20 @@
 # copyright 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, models
+from odoo import models
 
 
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
-    @api.multi
     def _prepare_payment_line_vals(self, payment_order):
         vals = super()._prepare_payment_line_vals(payment_order)
         if (
-            self.invoice_id
-            and self.invoice_id._is_isr_reference()
-            and self.invoice_id.partner_bank_id
+            self.partner_bank_id
+            and self.partner_bank_id._is_isr_issuer()
+            and self.move_id._has_isr_ref()
```

Comment:
From https://github.com/OCA/l10n-switzerland/pull/591#discussion_r675456232
> _is_isr_issuer checks if ISR postal code is valid.
>
> https://github.com/odoo/odoo/blob/14.0/addons/l10n_ch/models/res_bank.py#L32-L39

Why do you need to check if ISR postal code is valid if it was not needed in `12.0`?
If it is a new logic/fix please move it to a dedicated commit (or PR) so that the migration commit (or PR) only contains changes needed for the migration.